lib/rubygems.rb

```diff
@@ -846,6 +846,21 @@ def self.prefix
     end
   end
 
+  ##
+  # Git binary path
+
+  def self.git_path
```

Comment:
I think we might want to make this a private method in `environment_command.rb`, since that's the only place it is being used

Reply:
alright :+1:

Reply:
could call `send` in the tests ?